code/lib/cli/src/repro-templates.ts

```diff
@@ -21,9 +21,19 @@ export const allTemplates = {
       builder: '@storybook/builder-webpack5',
     },
   },
+  'nextjs/12-js': {
+    name: 'Next.js v12 (JavaScript)',
+    script:
```

Comment:
Did you want to add a TODO here to make a future PR to add `nextjs/12-js` to the `merged` array at the bottom once we have a template in repro-templates?

Reply:
I keep it in my notes. As soon as this PR is merged, I will immediately create a follow-up PR. :)